### ISSUE TITLE
Lite/Full split: finish preface

### ISSUE THREAD

**Issue by Ladicek** [eclipse-ee4j/cdi#490]
https://github.com/eclipse-ee4j/cdi/blob/864e1b4f779e4d6d5a80f147d975a510446d4987/spec/src/main/asciidoc/preface.asciidoc

**Comment by antoinesd** [comment]
Yes, I added that note. It was more to complete the history than to make it consistent. We can let it like that IMHO

**Comment by manovotn** [comment]
Ok, in that case, I'll remove the TODO and keep the document as it was.
